requirements/edx/base.txt

```diff
@@ -50,7 +50,7 @@ cryptography==2.9.2       # via -r requirements/edx/../edx-sandbox/shared.txt, d
 cssutils==1.0.2           # via pynliner
 ddt==1.3.1                # via -c requirements/edx/../constraints.txt, xblock-drag-and-drop-v2, xblock-poll
 decorator==4.4.2          # via pycontracts
-defusedxml==0.6.0         # via -r requirements/edx/base.in, djangorestframework-xml, ora2, python3-openid, python3-saml, safe-lxml, social-auth-core
+defusedxml==0.7.0rc1      # via -r requirements/edx/base.in, djangorestframework-xml, ora2, python3-openid, python3-saml, safe-lxml, social-auth-core
```

Comment:
In the last PR 
https://github.com/edx/edx-platform/commit/97494f89996463ead696018abc8a73fce1d035d9#diff-f9c61f9375a7917349c8217216293eb1 it was downgraded to 0.6.0, 

Do you know why your make upgrades returns 0.7.0rc1 and that one returned 0.6.0?